pkg/apis/componentconfig/types_pluginargs.go

```diff
@@ -78,3 +78,23 @@ type RemoveDuplicatesArgs struct {
 	Namespaces        *api.Namespaces
 	ExcludeOwnerKinds []string
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+type LowNodeUtilizationArgs struct {
+	metav1.TypeMeta
+
+	ThresholdPriority                 *int32
+	ThresholdPriorityClassName        string
```

Comment:
These are no longer validated at the arg/param level. It is now handled [here](https://github.com/kubernetes-sigs/descheduler/blob/83c4f5d5261e7113b6367c70914cedd1af7792ff/pkg/descheduler/descheduler.go#L371-L389)
```suggestion
```

Reply:
hi,  but i check the `GetPriorityFromStrategyParams` function, it still pass from `params` actually.

Reply:
I'm proposing that you no longer call `GetPriorityFromStrategyParams` since it's done at a higher level, therefore redundant.

Validation is now checked [here](https://github.com/kubernetes-sigs/descheduler/blob/83c4f5d5261e7113b6367c70914cedd1af7792ff/pkg/descheduler/descheduler.go#L371-L379)

And it's added to `EvictionFilter` [here](https://github.com/kubernetes-sigs/descheduler/blob/83c4f5d5261e7113b6367c70914cedd1af7792ff/pkg/descheduler/descheduler.go#L389)

Please use [this PR](https://github.com/kubernetes-sigs/descheduler/pull/913/files) as a reference. Note that it is no longer calling `GetPriorityFromStrategyParams` but all the e2e are still passing since it's part of `EvictionFilter`

Reply:
sorry english is my second lang, so sometimes i can not understand very well . but i will try to translate.  will update now